src/Importer/DBImport.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace TheRealMVP\Importer;
+
+class DBImport
+{
+    private \PDO $pdoConnection;
+    private array $apiData;
+
+    /**
+     * DBImport constructor - takes in a PDO connection and saves it locally to be used elsewhere in code
+     * Takes in a new GetAPI object, calls it's function getJson and stores the API Json as an array in local variable $apiData
+     *
+     * @param \PDO   $pdoConnection
+     * @param GetAPI $curlConnection
+     */
+    public function __construct(\PDO $pdoConnection, GetAPI $curlConnection)
+    {
+        $this->pdoConnection = $pdoConnection;
+        $this->apiData = $curlConnection->getJson();
```

Comment:
Would suggest that you just store the GetAPI object on a property (could just be called $api) instead of directly calling the getJson method in the constructor.

In the future, the API might be expanded to have other endpoints, in which case you would have to refactor this code so you don't call the existing endpoint every single time.

Then you can just call getJson at the start of storeData instead. Just makes it a little bit more flexible!